yarn.lock

```diff
@@ -1677,11 +1677,11 @@ ecc-jsbn@~0.1.1:
 
 "eclipse-che-operator@git://github.com/eclipse/che-operator#master":
   version "0.0.0"
-  resolved "git://github.com/eclipse/che-operator#2e47c3331782868d7b120718ded7be5f5e64c9da"
+  resolved "git://github.com/eclipse/che-operator#603f14a3d9c12b5c732870333787ec2748095ebb"
```

Comment:
are you sure that you need to update it within this PR?

Reply:
I don't see anything bad in depending on the latest

Reply:
I just don't understand what is a process of updating dependencies in yarn.lock, it's why I'm asking.